packages/demos/src/custom-logging/integrator.node.tsx

```diff
@@ -22,7 +22,7 @@ export default async function renderApp(): Promise<AppRendererResult> {
   });
 
   const html = ReactDOM.renderToString(
-    <FeatureHubContextProvider value={{featureAppManager}}>
+    <FeatureHubContextProvider value={{featureAppManager, logger}}>
```

Comment:
Please add this case to our 2.0 issue

Reply:
[done](https://github.com/sinnerschrader/feature-hub/issues/369)